docs/domain-model.md

```diff
@@ -13,56 +11,47 @@ Feature testing: Capybara (to be decided) / Travis CI
 
 Unit testing: RSpec, Ruby on Rails native testing framework
 
```

Comment:
Why deleting the line about Feature tests and Capybara?

Reply:
hi, I don't think they were deleted, I maybe took out a couple of lines and maybe the box formatting was changed but the two lines about hosting and feature are still in there :-)

Reply:
My bad, I didn't see it :)